### DISCUSSION TITLE
I cannot play MP3 on my page.

### DISCUSSION THREAD

**Discussion by turhanlil** [community/community#135417] [Programming Help] [answered]
### Select Topic Area

Question

### Body

Hey guys can you help me with audios in the html codes.

I couldnt manage to play music in my page. 

<audio id="background-music" autoplay loop>
    <source src="https://github.com/turhanlil/ilgi/blob/e2fa7c161de529e54b04704c405b2919a5502406/sarkimiz.mp3" type="audio/mpeg">
    Your browser does not support the audio element.
</audio>

This is the code. 

anything is wrong there?

**Comment by turhanlil** [comment]
`<audio id="background-music" autoplay loop>
    <source src="https://github.com/turhanlil/ilgi/blob/e2fa7c161de529e54b04704c405b2919a5502406/sarkimiz.mp3" type="audio/mpeg">
    Your browser does not support the audio element.
</audio>`

**Reply by davevad93** [reply]
Hi @turhanlil ,

Just replace it with this and it will work:

```html
<audio id="background-music" autoplay loop src="sarkimiz.mp3" type="audio/mp3"></audio>
```

**Reply by turhanlil** [reply]
I'll try but i see on network CORB is blocking my audio. How can i bypass that ?

**Reply by turhanlil** [reply]
Okay I dont see CORB on console network but.. There is no sound even though i put your code. Can you help me please?

**Reply by davevad93** [reply]
Hey @turhanlil , that's weird it should work because I tested changing the html in the browser dev tools and your music played automatically, but it's true I checked your page and now it doesn't work with my code.

**Reply by turhanlil** [reply]
Any solving way for this ?

**Reply by davevad93** [reply]
I don't know given that it should work this way, for example if you go to dev tools try deleting my code, paste it again and you'll see the sound works fine. I'll let you know if I find a solution.

**Reply by turhanlil** [reply]
Cannot see your code :p

**Reply by turhanlil** [reply]
The one i made it from visual code is working fine with your code. In the git hub i cant fix it.

**Reply by davevad93** [reply, marked as answer]
@turhanlil  I think the issue is that most browsers block audio being played by a web page without a user-triggered action. 
For more info see [here](https://developer.mozilla.org/en-US/docs/Web/Media/Autoplay_guide).

The solution would be to add an audio player to your page by simply adding the `controls` property in your `audio` tag.

![0](https://github.com/user-attachments/assets/3c7ef75e-5d9c-41e4-86df-d9a9c5fb387a)

If you press the play button, the music will play in a loop.

```html
<audio id="background-music" controls loop src="sarkimiz.mp3" type="audio/mp3"></audio>
```

**Comment by LiteBrite82** [comment]
Thanks for posting in the GitHub Community, @turhanlil ! 

We’ve moved your post to our Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.
